clang/docs/ReleaseNotes.rst

```diff
@@ -345,6 +345,9 @@ Bug Fixes to Compiler Builtins
 Bug Fixes to Attribute Support
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+- Clang now correctly matches plugin attributes with microsoft ``[attribute]`` syntax.
```

Comment:
Should this be double-brackets?
```suggestion
- Clang now correctly matches plugin attributes with Microsoft ``[attribute]`` syntax.
```

Reply:
Microsoft attributes use single brackets, c++11 attributes use `[[]]`